app.py

```diff
@@ -14,6 +15,46 @@ def index():
 def faq():
     return render_template('faq.html')
 
+
+@app.route('/signup', methods=['GET','POST'])
```

Comment:
You have multiple routes at the same time, 2 signup routes etc. Some will need to be deleted

Reply:
Should be fixed now